include/itkafDataManager.h

```diff
@@ -47,16 +46,14 @@ template< typename TBuffer >
 class DataManager: public Object
 {
 public:
-  typedef DataManager              Self;
+  using Self = DataManager;
   typedef Object                   Superclass;
```

Comment:
The other `typedef`s should be turned into `using`s as well.

Reply:
Done